tests/e2e/mapper/mapper_suite_test.go

```diff
@@ -0,0 +1,246 @@
+package mapper
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"path"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/api/core/v1"
+
+	"github.com/kubeedge/kubeedge/tests/e2e/utils"
+	. "github.com/onsi/ginkgo"
```

Comment:
please seperate vendor and golang packages

Reply:
Updated